index.bs

```diff
@@ -4073,8 +4081,13 @@ When registering a new credential, represented by an {{AuthenticatorAttestationR
         in <code>|authData|.[=attestedCredentialData=]</code>, as appropriate for the [=[RP]=]'s system.
     - Associate the <code>[=credentialId=]</code> with a new stored [=signature counter=] value
         initialized to the value of <code>|authData|.[=signCount=]</code>.
+    - OPTIONALLY, associate the <code>[=credentialId=]</code> with the transport hints
```

Comment:
it is a Good Catch to include this (thx!), tho I wonder whether we should have the "OPTIONALLY" qualifier here since it is RECOMMENDED....  Perhaps we ought to simply say:

It is RECOMMENDED to associate the <code>[=credentialId=]</code> with the transport hints returned by calling <code>|credential|.{{AuthenticatorAttestationResponse/getTransports()}}</code>, and to use these to populate the {{PublicKeyCredentialDescriptor/transports}} of the {{PublicKeyCredentialRequestOptions/allowCredentials}} option in future {{CredentialsContainer/get()}} calls to help the [=client=] know how to find a suitable [=authenticator=].

..since RECOMMENDED does allow for optionality (if ya got a good reason...).
￼

Reply:
> it is a Good Catch to include this (thx!)

Isn't this what #1188 was all about? :stuck_out_tongue_winking_eye:

>, tho I wonder whether we should have the "OPTIONALLY" qualifier here since it is RECOMMENDED....

Yeah, I would have liked to say something like "RECOMMENDEDLY, ..." but that isn't a proper RFC2119 term (and sounds stupid :smiley:). I edited this along the lines of your suggestion.

Reply:
OK, great, thanks.